x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,116 @@ export const PolicyList = React.memo(() => {
   const columns: Array<EuiTableFieldDataColumnType<Immutable<PolicyData>>> = useMemo(
     () => [
       {
-        field: 'name',
+        field: '',
         name: i18n.translate('xpack.siem.endpoint.policyList.nameField', {
           defaultMessage: 'Policy Name',
         }),
         // eslint-disable-next-line react/display-name
-        render: (value: string, item: Immutable<PolicyData>) => {
+        render: (item: Immutable<PolicyData>) => {
           const routePath = getManagementUrl({
             name: 'policyDetails',
             policyId: item.id,
             excludePrefix: true,
           });
           const routeUrl = getManagementUrl({ name: 'policyDetails', policyId: item.id });
-          return <PolicyLink name={value} route={routePath} href={routeUrl} />;
+          return (
+            <>
+              <EuiFlexItem grow={false}>
+                <PolicyLink name={item.name} route={routePath} href={routeUrl} />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiText
+                  color="subdued"
+                  size="xs"
+                  style={{ marginLeft: '6px', whiteSpace: 'nowrap' }}
+                >
+                  <FormattedMessage
+                    id="xpack.siem.endpoint.policyList.revision"
+                    defaultMessage="rev. {revNumber}"
+                    values={{ revNumber: item.revision }}
+                  />
+                </EuiText>
+              </EuiFlexItem>
+            </>
+          );
         },
         truncateText: true,
```

Comment:
FYI - this is no longer doing anything. Whenever the `render: ()=>{}` field definition prop is defined, the trancate gets overriden back to "false".  You will need to provide your own way to truncate values in this case (this applies to any other change here where `render()` is defined). The good news is that in most cases, its as easy as adding `eui-textTruncate` to your element (see [Eui CSS Utilities](https://elastic.github.io/eui/#/utilities/css-utility-classes) )